docs/pages/world-explorer.mdx

```diff
@@ -0,0 +1,163 @@
+  <Tab>
+    <iframe
+      width="761"
+      height="523"
+      src="https://www.youtube.com/embed/87wuGFH9Pz0"
+      title="Using the MUD Data Explorer"
+      frameborder="0"
+      allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
+      referrerpolicy="strict-origin-when-cross-origin"
+      allowfullscreen
+    ></iframe>
+  </Tab>
+  <Tab>
+    <iframe
+      width="761"
+      height="523"
+      src="https://www.youtube.com/embed/87wuGFH9Pz0?t=281https://www.youtube.com/embed/87wuGFH9Pz0?si=EsgcYGK9qk6J-Xvs&amp;start=281"
+      title="Using the MUD Data Explorer's Interact tab"
+      frameborder="0"
+      allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
+      referrerpolicy="strict-origin-when-cross-origin"
+      allowfullscreen
+    ></iframe>
+  </Tab>
+</Tabs>
+
+### CLI options
+
+To use the explorer with different command-line options, use this process:
+
+1. In the mprocs screen, go down to the **explorer** process.
+
+1. Type `x` to stop the default explorer.
+
+1. In a different command-line window, go to `packages/contract`.
+
+1. Run the explorer using `pnpm explorer <options>`.
+
+The World Explorer accepts the following CLI options:
+
+| Option              | Description                                                         | Default value |
+| ------------------- | ------------------------------------------------------------------- | ------------- |
+| `--worldAddress`    | The address of the world to explore                                 | None          |
+| `--worldsFile`      | Path to a worlds configuration file (used to resolve world address) | worlds.json   |
+| `--indexerDatabase` | Path to your SQLite indexer database                                | indexer.db    |
+| `--chainId`         | The chain ID of the network                                         | 31337         |
+| `--port`            | The port on which to run the World Explorer                         | 13690         |
+| `--dev`             | Run in development mode                                             | false         |
+
+## Installation (pre-2.2 MUD)
+
+1. Install the Indexer and World Explorer packages in the project root.
+
+   ```sh copy
+   pnpm install @latticexyz/store-indexer@latest -w
+   pnpm install @latticexyz/explorer@latest -w
+   ```
+
+1. Edit `mprocs.yaml` to add the `indexer` and `explorer` processes.
+
+   <CollapseCode>
+
+   ```yaml filename="mprocs.yaml" copy showLineNumbers {11-20}
+   procs:
+     client:
+       cwd: packages/client
+       shell: pnpm run dev
+     contracts:
+       cwd: packages/contracts
+       shell: pnpm mud dev-contracts --rpc http://127.0.0.1:8545
+     anvil:
+       cwd: packages/contracts
+       shell: anvil --base-fee 0 --block-time 2
+     indexer:
+       cwd: packages/contracts
+       shell: rimraf $SQLITE_FILENAME && pnpm sqlite-indexer
```

Comment:
```suggestion
       shell: rm -rf $SQLITE_FILENAME && pnpm sqlite-indexer
```
changed this to a regular `rm -rf` instead of `rimraf` or `shx` equivalents since we won't know which env folks will be in when adding this